components/ProfilePage/FollowingTab.tsx

```diff
@@ -76,55 +104,95 @@ export function FollowingTab({ className }: Props) {
   })
 
+  const orgsFollowingQuery = async () => {
+    const q = query(
+      subscriptionRef,
+      where("uid", "==", `${uid}`),
+      where("type", "==", "org")
+    )
+    const querySnapshot = await getDocs(q)
+    querySnapshot.forEach(doc => {
+      // doc.data() is never undefined for query doc snapshots
+      orgsList.push(doc.data().orgId)
+    })
+
+    if (orgsFollowing.length === 0 && orgsList.length != 0) {
+      setOrgsFollowing(orgsList)
+    }
+  }
+
+  useEffect(() => {
+    uid ? orgsFollowingQuery() : null
+  })
+
+  const onSubmit = async () => {
+    console.log("submitted")
+  }
+
   return (
     <>
       <TitledSectionCard className={className}>
         <div className={`mx-4 mt-3 d-flex flex-column gap-3`}>
           <Stack>
             <h2>Bills You Follow</h2>
-            {billsFollowing.map((element: any) => (
-              <Styled key={element}>
-                <External
-                  href={`https://malegislature.gov/Bills/${element.court}/${element.bill}`}
-                >
-                  {formatBillId(element.bill)}
-                </External>
-                <Row>
-                  <Col className={`col-10`}>
-                    <BillFollowingTitle
-                      court={element.court}
-                      id={element.bill}
-                    />
-                  </Col>
-                  <Col
-                    className={`text-center`}
-                    onClick={() => {
-                      setUnfollowModal("show")
-                      setCurrentBill(element.bill)
-                    }}
-                  >
-                    <button
-                      className={`btn btn-link d-flex align-items-start p-0 text-decoration-none`}
-                    >
-                      <h6>Unfollow</h6>
-                    </button>
-                  </Col>
-                  <hr className={`mt-3`} />
-                </Row>
-              </Styled>
+            {billsFollowing.map((element: any, index: number) => (
+              <FollowedBill
+                key={index}
+                element={element}
+                setCurrentCourt={setCurrentCourt}
+                setCurrentOrgName={setCurrentOrgName}
+                setCurrentType={setCurrentType}
+                setCurrentTypeId={setCurrentTypeId}
+                setUnfollowModal={setUnfollowModal}
+              />
             ))}
           </Stack>
         </div>
       </TitledSectionCard>
-      <TitledSectionCard className={`${className} invisible`}>
+      <TitledSectionCard className={`${className}`}>
         <div className={`mx-4 mt-3 d-flex flex-column gap-3`}>
-          <Stack>
-            <h2>Organizations You Follow</h2>
-          </Stack>
+          <Styled>
+            <Row>
+              <Col>
+                <h2 className={``}>Organizations You Follow</h2>
+              </Col>
+              <Col className={`d-flex invisible justify-content-end`}>
```

Comment:
Let's remove this -- the figma says we're punting the search bar to Q2

Reply:
done